invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/details/meta.html

```diff
@@ -12,7 +12,7 @@
 
 {%- set meta_title = record.metadata.title|striptags -%}
 {%- set meta_description = record.metadata.description|striptags -%}
-{%- set meta_authors = record.creators|map(attribute='name') -%}
+{%- set meta_authors = record.metadata.creators|map(attribute='person_or_org')|map(attribute='name') -%}
```

Comment:
Should we filter out organizations, or is it fine to include both people and organizations?